meetings/2017/05-18.md

```diff
@@ -19,11 +22,330 @@ Attendees
+  static get is(){ return 'my-cmp' }
+  static get props () {
+    return {
+      myArray: prop.array(),
+      myBoolean: prop.boolean()
+    }
+  }
+}
+
+define(MyComponent)
+```
+
+**5.x**
+
+```js
+import { h, Component, props } from 'skate'
+ 
+class MyComponent extends Component {
+  static get is(){ return 'my-cmp' }
+  static get props () {
+    return {
+      myArray: props.array,
+      myBoolean: props.array
+    }
+  }
+}
+
+define(MyComponent)
+```
+
+### Properties API => get/set are gone from prop API
+
+// Comparison TODO
+
+### Properties API => setting/getting props ( triggering re-render )
+
+**4.x**
+
+```js
+// MyComponent.jsx
+import { h, Component, prop, props } from 'skate'
+ 
+class MyComponent extends Component {
+  static get is(){ return 'my-cmp' }
+  static get props () {
+    return {
+      myArray: prop.array(),
+      myBoolean: prop.boolean()
+    }
+  }
+  someNonPublicApiProp = 'Who are you?'
+
+  _changeProps(){
+
+    props(this,{myBoolean: true})
+    // or just directly
+    this.myBoolean = true
+    
+    console.log(props(this)) // { myArray: [], myBoolean: true }
+
+    props( this, {myArray: ['hello']} )
+      // or just directly
+    this.myArray = ['hello']
+
+    console.log(props(this)) // { myArray: ['hello'], myBoolean: true }
+
+    // this will not trigger re-render
+    this.someNonPublicApiProp = 'Im David'
+  }
+}
+
+define(MyComponent)
+```
+
+**5.x**
+
+```js
+// MyComponent.jsx
+import { h, Component, props } from 'skate'
+ 
+class MyComponent extends Component {
+  static get is(){ return 'my-cmp' }
+  static get props () {
+    return {
+      myArray: props.array,
+      myBoolean: props.boolean
+    }
+  }
+  someNonPublicApiProp = 'Who are you?'
+
+  _changeProps(){
+    this.props = {myBoolean: true}
+    // or just directly
+    this.myBoolean = true
+    
+    console.log(this.props) // { myArray: [], myBoolean: true }
+
+    this.props = { myArray: ['hello'] }
+      // or just directly
+    this.myArray = ['hello']
+
+    console.log(this.props) // { myArray: ['hello'], myBoolean: true }
+
+    // this will not trigger re-render
+    this.someNonPublicApiProp = 'Im David'
+  }
+}
+
+define(MyComponent)
+```
+
+
+### Defining Custom Renderer
+
+**4.x**
+
+NOT POSSIBLE
+
+**5.x**
+// TODO add example
+
+
 ## Flow
 
 - Flow can be merged sooner rather than later if we can get `get-typed` published.
 
+- need explanation of TS generated files by Flow secret tool before merging as LGTM types, see -> https://github.com/skatejs/skatejs/pull/1163#discussion_r116605816
+
 ## ES latest build
 
 - Remove `module` in package.json.
 - Merge es latest build.
+
+
+## @hotell questions
+
+- since `webcomponents.js` **1.0** is out ( YES ALSO on NPM finally !!!) we can drop our custom skatejs/webcomponents ? ( our custom ones doesnt work with Angular )
+
+- what will/is the new size skate 5.x + preact 8.x as renderer ( marketing )
+
+- what type of source are we gonna ship ?
```

Comment:
Yeah, okay. I think that's simple enough to provide because all we need to do is:

1. Re-enable the `esModule` build with NWB
2. Change the `es/` output directory for the Babel step to `es-next/` or `es-latest/`.
3. Bikeshed the name of the new directory because it becomes API.